### PR TITLE
feat: add distributed rolling update tests to 2.0

### PR DESCRIPTION
**Pull request by JoanFM**

### CODE REVIEW

File: jina/flow/__init__.py [hunk 1]
```diff
@@ -7,3 +7,16 @@ class Flow(PostMixin, BaseFlow):
 
     For proper usage see `this guide` <https://docs.jina.ai/chapters/flow/index.html>
     """
+
+    def dump(self, pod_name: str, dump_path: str, shards: int, timeout=-1):
```
Comment (hanxiao) [inline jina/flow/__init__.py:11]: besides due to the legacy code, why dump qualify to be here?

**Comment by hanxiao** [comment]
please on hold this PR for now

**Comment by hanxiao** [comment]
please on hold this PR for now

**Comment by JoanFM** [comment]
@hanxiao ,

I know the `dump` presence is not clear, but the test is working, I think it would be worth adding it

**Comment by hanxiao** [comment]
https://github.com/jina-ai/jina/pull/2421

**Pull request merged**